style.css

```diff
@@ -250,7 +300,56 @@ body {
   margin: 0 16px 0 0;
 }
 
-@media screen and (min-width: 375px) {
+
+@media screen and (max-width: 768px) {
+
+  .navBar {
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 10;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding-top: 100px;
+    align-items: flex-start;
+    padding-left: 30px;
+    background: #6070ff;
+    color: #ebebff;
+    opacity: 0.9;
+    transition:top 1s ease;
+    display: none;
```

Comment:
Kindly consider including the background blur effect to replicate the Figma design completely. You can use the following code as a reference: backdrop-filter: blur(5px). Achieving the same result using alternative methods is also possible. 😁